### PR TITLE
boost is this you search

### PR DESCRIPTION
**Pull request by DanielPalafox**
The PR https://github.com/ORCID/ORCID-Source/pull/6255 must be merged as well

### CODE REVIEW

**Comment by DanielPalafox** [comment]
[Boost records with affiliations in the "Could this be you?" modal](https://trello.com/c/XrHvdPDG/14-boost-records-with-affiliations-in-the-could-this-be-you-modal)

**Review by leomendoza123 — APPROVED** [review]

File: src/app/core/search/search.service.ts [hunk 1]
```diff
@@ -33,24 +33,25 @@ export class SearchService {
...
 
-  private buildSearchUrl(querryParam: SearchParameters): string {
+  public buildSearchUrl(queryParam: SearchParameters): string {
     const escapedParams: SearchParameters = {}
-    Object.keys(querryParam).map((key) => {
-      escapedParams[key] = this.escapeReservedChar(querryParam[key])
+    Object.keys(queryParam).map((key) => {
+      escapedParams[key] = this.escapeReservedChar(queryParam[key])
     })
+    console.log(JSON.stringify(escapedParams))
```
Comment (leomendoza123) [inline src/app/core/search/search.service.ts:41]: PLeas review the console logs

**Review by leomendoza123 — CHANGES_REQUESTED** [review]

File: src/app/core/search/search.service.ts (hunk 1 above)
Comment (DanielPalafox) [reply src/app/core/search/search.service.ts]: Done @leomendoza123

**Review by leomendoza123 — APPROVED** [review]

**Comment by amontenegro** [comment]
We are going to use the same components as the UI search, so, we can close this PR

**Pull request closed without merging**